lang/de.json

```diff
@@ -741,6 +741,26 @@
         "ShareaholicTagSiteIdDescription": "Erstellen Sie eine neue Seite in Shareaholic und setzen Sie die Site-ID hier ein",
         "ShareaholicTagSiteIdTitle": "Shareaholic Site-ID",
         "ShowAdvancedSettings": "Erweiterte Einstellungen anzeigen",
+
+        "SiteWithoutDataReactIntro": "Das Verfolgen Ihrer React.js-Website oder -App ist mit Matomo einfach, selbst für Single-Page-Anwendungen.",
+        "SiteWithoutDataReactFollowStepCompleted": "%1$sHerzlichen Glückwunsch!%2$s Sie haben den Matomo Analytics-Tracking-Code erfolgreich über den Matomo Tag Manager installiert. Überprüfen Sie zur Bestätigung, dass die Treffer erfasst werden, Ihre Website und prüfen Sie, ob diese Daten in Ihrer Matomo-Instanz sichtbar sind.",
+        "ReactFollowStep16": "Fügen Sie den %1$sMatomo Tag Manager JS%2$s-Code in Ihre %3$s (oder andere relevante Dateien) ein. Wir empfehlen, dies mit der Methode \"%4$s\" zu tun. Das folgende Beispiel zeigt, wie Sie den %1$sMatomo Tag Manager JS%2$s-Code in einer \"%5$s\"-App in %6$s hinzufügen.",
+
+        "SPAFollowStep1": "Wählen Sie den richtigen Container für Ihre Website aus.%1$sHinweis:%2$s Matomo konfiguriert automatisch einen Container, wenn Sie eine Website hinzufügen. Standardmäßig ist dieser Container in den unten stehenden Feldern ausgewählt. Wenn Sie einen einzelnen Container für mehrere Websites verwenden möchten, stellen Sie sicher, dass Sie diesen spezifischen Container auswählen. Sie können %3$shier Ihre Container verwalten%4$s.",
+        "SPAFollowStep2": "Navigieren Sie zu den %1$sTrigger des Containers%2$s.",
+        "SPAFollowStep3": "Überprüfen Sie, ob bereits ein Trigger vom Typ %1$s%5$s%2$s vorhanden ist. Wenn Sie den %5$s-Trigger nicht sehen, folgen Sie %3$sdieser Anleitung%4$s.",
```

Comment:
Is there a reason for the placeholder complexity? Can't we use one placeholder instead of `%1$s%5$s%2$s`? This makes it less error prone for translators. Same also applies for other strings below.

Reply:
It is a combination of pulled-in translated text from other parts of the UI and it being wrapped in `<strong></strong>` or `<a href...></a>` tags.

I guess we can hide this complexity through a single placeholder and do the wrapping in the component itself.

Reply:
These translations also need to be marked as Reviewed for the top languages but I guess in other languages we still want the community to help us translate it..

Reply:
Ok, on it.

Reply:
@sgiehl done.